Khan/khan-linter
# Add eslint-comments plugin

This plugin has a couple of rules that we'd like to use.  In particular:
- no-unused-disable
- no-unlimited-disable

Test Plan:
- in webapp's .eslintrc
  - add "eslint-comments" to the plugin section
  - enable the "no-unused-disable" plugin
- run 'ka-lint javascript'
- see that there are 141 unused disable comments
